### DISCUSSION TITLE
thanks/

### DISCUSSION THREAD

**Discussion by giscus** [GZTimeWalker/blog-comments#22] [Announcements]
# thanks/

致谢  本人部分项目目前接受捐赠，扫描下方二维码可以投食促进项目进展，请备注 “捐赠” 及自己的信息。 目前接受捐赠的项目：GZCTF 捐赠二维码：      捐赠列表 金额较大的捐赠者会在这里进行记录以示感谢。    捐赠者 金额 时间 备注     极核 ¥50.00 2024/04/16 这周四我 v 你   ? ¥6.66 2024/04/16    江西软件职业技术大学 ¥100.00

https://blog.gzti.me/thanks/

**Comment by RIeFre** [comment]
太棒了
